ext/mbstring/mbstring.stub.php

```diff
@@ -20,37 +20,37 @@ function mb_parse_str(string $encoded_string, &$result): bool {}
+function mb_strlen(string $str, ?string $encoding = null): int {}
 
-function mb_strpos(string $haystack, string $needle, int $offset = 0, string $encoding = UNKNOWN): int|false {}
+function mb_strpos(string $haystack, string $needle, int $offset = 0, ?string $encoding = null): int|false {}
 
-function mb_strrpos(string $haystack, string $needle, int $offset = 0, string $encoding = UNKNOWN): int|false {}
+function mb_strrpos(string $haystack, string $needle, int $offset = 0, ?string $encoding = null): int|false {}
 
-function mb_stripos(string $haystack, string $needle, int $offset = 0, string $encoding = UNKNOWN): int|false {}
+function mb_stripos(string $haystack, string $needle, int $offset = 0, ?string $encoding = null): int|false {}
 
-function mb_strripos(string $haystack, string $needle, int $offset = 0, string $encoding = UNKNOWN): int|false {}
+function mb_strripos(string $haystack, string $needle, int $offset = 0, ?string $encoding = null): int|false {}
 
-function mb_strstr(string $haystack, string $needle, bool $part = false, string $encoding = UNKNOWN): string|false {}
+function mb_strstr(string $haystack, string $needle, bool $part = false, ?string $encoding = null): string|false {}
 
-function mb_strrchr(string $haystack, string $needle, bool $part = false, string $encoding = UNKNOWN): string|false {}
+function mb_strrchr(string $haystack, string $needle, bool $part = false, ?string $encoding = null): string|false {}
 
-function mb_stristr(string $haystack, string $needle, bool $part = false, string $encoding = UNKNOWN): string|false {}
+function mb_stristr(string $haystack, string $needle, bool $part = false, ?string $encoding = null): string|false {}
 
-function mb_strrichr(string $haystack, string $needle, bool $part = false, string $encoding = UNKNOWN): string|false {}
+function mb_strrichr(string $haystack, string $needle, bool $part = false, ?string $encoding = null): string|false {}
 
-function mb_substr_count(string $haystack, string $needle, string $encoding = UNKNOWN): int {}
+function mb_substr_count(string $haystack, string $needle, ?string $encoding = null): int {}
 
-function mb_substr(string $str, int $start, ?int $length = null, string $encoding = UNKNOWN): string {}
+function mb_substr(string $str, int $start, ?int $length = null, ?string $encoding = null): string {}
 
-function mb_strcut(string $str, int $start, ?int $length = null, string $encoding = UNKNOWN): string|false {}
+function mb_strcut(string $str, int $start, ?int $length = null, ?string $encoding = null): string|false {}
 
-function mb_strwidth(string $str, string $encoding = UNKNOWN): int {}
+function mb_strwidth(string $str, ?string $encoding = null): int {}
 
-function mb_strimwidth(string $str, int $start, int $width, string $trim_marker = UNKNOWN, string $encoding = UNKNOWN): string {}
+function mb_strimwidth(string $str, int $start, int $width, ?string $trim_marker = null, ?string $encoding = null): string {}
```

Comment:
trim_marker is not nullable in the implementation. It should be possible to default it to `""` though.